src/utilities/router/Router.tsx

```diff
@@ -64,6 +79,16 @@ function _getComponent(matchPath, children) {
   return null;
 }
 
+function _hasAnchorLink(path) {
+  return (path.match(/#/g) || []).length > 1;
+}
+
+function _extractBasePath(path) {
+  let split = path.split('#');
```

Comment:
Not sure this will work correctly because this will just return nothing as indexOf will always return 0 because the pound sign exists at the beginning of  path.